File: runtime/Jlib/unspecified.java
package bigloo;

public final class unspecified extends cnst
{
  public static final unspecified unspecified = new unspecified( 3 );

  private unspecified( final int  n )
  {
    super( n );
  }

  public void write( final output_port  p )
  {
    p.write( "#unspecified" );
  }
}

File: runtime/CSlib/unspecified.cs
namespace bigloo 
{
  public sealed class unspecified: cnst
  {
    public static readonly unspecified _unspecified= new unspecified( 3 );

    private unspecified( int  n )
      : base( n )
    {
    }

    public override void write( output_port  p )
    {
      p.write( "#unspecified" );
    }
  }
}
